### PR TITLE
Feat  skip to content

### PR DESCRIPTION
**Pull request by theogainey**
### Description
This PR adds a SkipToContent component, a visually hidden mixin, and base styles to SkipToContent component so that it is  visually hidden but appears on focus. 

### Spec

See Story: [FSA22V1-171](https://sparkbox.atlassian.net/browse/FSA22V1-171)

### Validation

* [ ] This PR has code changes, and our linters still pass.
* [ ] This PR affects production code, so it was browser tested (see below).
* [ ] Changes are browser tested. This includes functionality, accessibility, responsiveness, and design.

#### To Validate

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, Snyk, etc).
2. Pull down all related branches.
3. Confirm all tests pass.
4. Open preview deploy and confirm that SkipToContent component is initially visually hidden, can be navigated to with the tab key, appears when in focus, and switches focus when clicked

### CODE REVIEW

File: pages/index.jsx
Comment (theogainey) [inline pages/index.jsx:13]: I am wondering if using `useRef` and the DOM method `.focus()` is necessary. I was having trouble getting focus to switch without it but I also initially had a setting turned on in my browser that affected tabbing through a page.

**Review by dustin-jw — CHANGES_REQUESTED** [review]
Sometimes when all you have is a hammer (React), every problem looks like a nail (a use case for `useRef`) 😂 

This can be simplified a bit, and extended to other pages, but you've got the basics in place.

File: styles/globals.scss
Comment (dustin-jw) [inline styles/globals.scss:78]: This probably got obliterated in conflict resolution, but these styles were removed in an earlier PR, so we should get rid of them here, too.

File: styles/globals.scss
Comment (dustin-jw) [inline styles/globals.scss:100]: Lets move this to `styles/components/_skip-to-content.scss` and then import that file here. Ideally, the entry point should be composed entirely of `@import` statements.

I'd like to see the class name namespaced as well, like `.cmp-skip-to-content` ('cmp' for component).

**Review by dustin-jw — CHANGES_REQUESTED** [review]
Changes look good! I just noticed a few CSS things that we can clean up before merging.

**Comment by theogainey** [comment]
At the risk of getting ahead of myself I am going to go ahead and ask @arnest00 to come in and give it the second review.

**Review by dustin-jw — APPROVED** [review]

File: styles/globals.scss
Comment (arnest00) [inline styles/globals.scss:1]: Thanks for getting this file organized like this!

**Pull request merged**
